tests/actions/PolicyTagTest.ts

```diff
@@ -642,7 +644,13 @@ describe('actions/Policy', () => {
     });
 
     describe('DeletePolicyTags', () => {
-        it('delete policy tag', () => {
+        it('should return early if no policyTag is found', () => {
+            const fakePolicy = createRandomPolicy(0);
+            const result = deletePolicyTags(fakePolicy.id, ['tag1'], {});
```

Comment:
Do we have to create the fake policy here? 🤔 Can't we just pass some random string?

Reply:
Thanks, you're right.

And you actually gave me the idea that this test isn't the best.
I'm replacing it with two others:

- should not modify Onyx data when policyTags is empty
- should not modify Onyx data when tagsToDelete do not exist